test/Microsoft.AspNetCore.WebSockets.Test/KestrelWebSocketHelpers.cs

```diff
@@ -38,7 +38,14 @@ public static IDisposable CreateServer(ILoggerFactory loggerFactory, Func<HttpCo
                         await ct.Response.WriteAsync(ex.ToString());
                     }
                 });
-                builder.UseWebSockets();
+                if (options == null)
```

Comment:
You don't need this conditional anymore, replace it with AddWebSockets

Reply:
Wouldn't AddWebSockets look even worse and it would need to be updated if anything is added.
```
services.AddWebSockets(o =>
{
    o.AllowedOrigins = options.AllowedOrigins;
    o.KeepAliveInterval = options.KeepAliveInterval;
    o.ReceiveBufferSize = options.ReceiveBufferSize;
});
```

Reply:
Not if you change CreateServer to take a lambda with a default that no-ops.